kodilla-advanced-tests/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.3.1'
     testImplementation("org.junit.jupiter:junit-jupiter-params:5.4.2")
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.3.1'
+    compile group: 'com.google.guava', name: 'guava', version: '29.0-jre'
```

Comment:
z tego co widze, to w koncu uzylas `Set.of ` z Javy 9. Guava jest w takim razie niepotrzebna (guava dawala api `Sets.newHashset(...)`